src/Bridge/Symfony/Bundle/SonataTwigExtensionsBundle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Sonata\TwigExtensions\Bridge\Symfony\Bundle;
+
+use Sonata\TwigExtensions\Bridge\Symfony\DependencyInjection\Compiler\StatusRendererCompilerPass;
+use Sonata\TwigExtensions\Bridge\Symfony\DependencyInjection\SonataTwigExtensionsExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class SonataTwigExtensionsBundle extends Bundle
+{
+    public function getPath()
```

Comment:
I think you can add type hints

Reply:
I added them somewhere, somewhere forgot, will continue tomorrow :smile: